inscriptions/ISic002999.xml

```diff
@@ -193,60 +198,31 @@
-                    ἐπὶ ἀμφιπόλου ἐ<supplied reason="lost">ν Συρακού</supplied>σαις<lb n="4"/>
-                    <expan>
-                        <abbr>Α</abbr>
-                        <ex>ὔλου</ex>
-                    </expan>
-                    Ματίου Σέστου, ἱερείας δὲ<lb n="5"/>τᾶν Παίδων Καικιλίας,<lb n="6"/>οἱ παραγενόμενοι ποτὶ
-                    Νύ<supplied reason="omitted">μ</supplied>φας<lb n="7"/>καὶ εὐχαριστήσαντες, μηνὸς<lb n="8"/>
-                    Ἀπελλαίου τρίτᾳ, Σέξτος<lb n="9"/>
-                    <expan>
-                        <abbr>Α</abbr>
-                        <ex>ὖλος</ex>
-                    </expan>
-                    
-                    ∙ Μάρκις Λόν<supplied reason="lost">γος</supplied>,<lb n="10"/>
-                    <expan>
-                        <abbr>Κ</abbr>
-                        <ex>όϊντος</ex>
-                    </expan>
-                    ∙ Ὄππις Κάνακις,<lb n="11"/>
-                    <expan>
-                        <abbr>Κ</abbr>
-                        <ex>όϊντος</ex>
-                    </expan>
-                    ∙ Ἄμπιους Κρήσκη<supplied reason="lost">ς</supplied>,<lb n="12"/> ․ Αἴλις Σελλα
-                    <supplied reason="lost">— — — — —</supplied>
-                    <lb n="13"/>
-                    <expan>
-                        <abbr>Κ</abbr>
-                        <ex>όϊντος</ex>
-                    </expan>
-                    <supplied reason="lost">— —</supplied>ήσις <supplied reason="lost">—</supplied>τιανός,<lb n="14"/>
-                    
-                    <expan>
-                        <abbr>Γ</abbr>
-                        <ex>άϊος</ex>
-                    </expan>
-                    <supplied reason="lost">— —</supplied>ιφ<supplied reason="lost">— — —</supplied>κρατος<lb n="15"/>
-                    
-                    <expan>
-                        <abbr>Γ</abbr>
-                        <ex>άϊος</ex>
-                    </expan>
-                    <supplied reason="lost">— — — — — —</supplied>μητος.
-                    <lb n="16"/>
+                    <lb n="1"/>χαριστήρια Παίδεσσι καὶ
+                    <lb n="2"/>Ἄννᾳ <unclear>κ</unclear><supplied reason="lost">αὶ</supplied> <supplied reason="lost">Ἀπ</supplied>όλ<reg>λ</reg>ωνι
+                    <lb n="3"/>ἐπὶ ἀμφιπόλου ἐ<supplied reason="lost">ν</supplied> <supplied reason="lost">Συρακούσ</supplied>αις <gap reason="lost" quantity="1" unit="character"/>
+                    <lb n="4"/><orig>ΜΙ</orig><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">ἱερείας</supplied> δὲ
+                    <lb n="5"/>τᾶν Παίδων Καικιλίας
+                    <lb n="6"/>οἱ παραγενόμενοι ποτὶ Νύμφας
+                    <lb n="7"/>καὶ εὐχαριστήσαντες μηνὸς
+                    <lb n="8"/>Ἀπελλαίου τρίτᾳ
+                    <lb n="9"/><abbr>Λ</abbr> <g type="interpunct">·</g> Μάρκις Λόνγος
```

Comment:
not necessary, but do we feel brave enough to expand the Greekv ersions of these praenomina?